alphagov/paas-product-page
# [#160508812] UK hosting

## What

Announce the UK hosting

## How to review

Run it locally: `make dev`.

* check links work
* proofread
* check the pricing-detailed page isn't used by anything
* confirm it is okay for us to announce public beta of Elasticsearch. I think this is okay, but I am only 96.4% confident.

## Who

Anyone but me.
